frontend/packages/console-dynamic-plugin-sdk/src/extensions/create-resource.ts

```diff
@@ -0,0 +1,30 @@
+import { match as RouterMatch } from 'react-router';
+import { ExtensionK8sModel } from '../api/common-types';
+import { CodeRef, Extension, ExtensionDeclaration } from '../types';
+
+export type CreateResource = ExtensionDeclaration<
+  'console.resource/create',
+  {
+    /** The model for which this create resource page will be rendered. */
+    model: ExtensionK8sModel;
+    /** The component to be rendered when the model matches */
+    component: CodeRef<
+      React.ComponentType<{
+        match: RouterMatch<{
+          /** Current namespace */
+          ns?: string;
+          /**
+           * Name of the OLM operator.
+           * Defined if the page is opened from the OLM installed operators page.
+           */
+          appName?: string;
+        }>;
+      }>
+    >;
+  }
+>;
+
+// Type guards
+
```

Comment:
nit, extra newline

Reply:
This follows style of all other extension declarations. We always add the newline after `// Type guards` comment